### ISSUE TITLE
Unlicensed assets

### ISSUE THREAD

**Issue by bkeys** [KranX/Vangers#103]
I looked around on my local copy of Vangers and could not find a license for the assets. Could you tell me what license the game assets are under?

**Comment by stalkerg** [comment]
Hello, it's standard commercial license (I think from Steam). 
You can't redistribute and commercial usage.

**Comment by DiTolk** [comment]
Думаю можно закрыть

**Comment by throwaway1037** [comment]
Please may the files be released under a [libre](https://freedomdefined.org/Licenses) license?
Some OS distributions such as [Parabola](https://parabola.nu) cannot package games which are not libre.

**Comment by throwaway1037** [comment]
Please can I convince you that the files should be liberated?
There is no point instating an artificial "balance" between proprietary and libre, when all works should be libre.

**Comment by bkeys** [comment]
I have to agree with @throwaway1037 there is no real reason to keep the assets proprietary. That was my intention of making this issue in the first place. It would also help people like myself who would make a package and submit it for inclusion in the Fedora repositories, but am unable to because of the proprietary artwork.

**Comment by throwaway1037** [comment]
Thank you, @bkeys.
In effect, this project is crippled in the free world because it is not entirely libre.
The code being libre is a great positive step, and for that I think the authors, but without all the components being libre, the project as a whole is unusable in freedom.

**Comment by caiiiycuk** [comment]
This is commercial project why it should be free? I don’t understand.

**Comment by throwaway1037** [comment]
The word "free" is ambiguous in the English language; it can refer to either price (as in "free beer") or freedom (as in "free speech").
I am referring to the latter meaning: freedom.

**Comment by caiiiycuk** [comment]
Yeah but this should decide authors of art works. We are not have such rights, however you can play full version in [browser](https://www.gamepix.com/play/vangers) it also have community mods like full 3D version

**Comment by throwaway1037** [comment]
> Yeah but this should decide authors of art works. We are not have such rights

I am sorry, I don't know what you mean.
Do you mean that you are not the copyright holder of the proprietary files?

**Comment by throwaway1037** [comment]
Do you know who they are?

**Comment by Fenex** [comment]
Usually companies release the code of old games under an open source due to get free (no money) support running a game on modern hardware and OS (by game's fans). The reason is not to believe in freedom ))

This game is still sell so it's pretty strange to do what you want.

**Comment by throwaway1037** [comment]
I agree the authors of the game should be able to make money on the game.
Releasing the proprietary data under a libre license doesn't stop this.
